File: KrasiMilan/QL/src/main/java/nl/uva/softwcons/ast/expression/binary/comparison/GreaterThan.java
package nl.uva.softwcons.ast.expression.binary.comparison;

import nl.uva.softwcons.ast.LineInfo;
import nl.uva.softwcons.ast.expression.Expression;
import nl.uva.softwcons.ast.expression.ExpressionVisitor;

public class GreaterThan extends ComparisonExpression {

    public GreaterThan(final Expression left, final Expression right, final LineInfo lineInfo) {
        super(left, right, lineInfo);
    }

    @Override
    public <T> T accept(ExpressionVisitor<T> visitor) {
        return visitor.visit(this);
    }
}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expression/Binary/GreaterThan.cs
using AST.Nodes.Interfaces;
using AST.Representation;

namespace AST.Nodes.Expression.Binary
{
    public class GreaterThan : Binary, IExpression
    {
        
        public GreaterThan(IExpression left, IExpression right, PositionInText position)
            : base(left, right, position)
        {}

        public override T Accept<T>(ASTVisitors.IVisitor<T> visitor)
        {
            return visitor.Visit(this);
        }

        public override string ToString()
        {
            return ">";
        }
    }
}
